lib/Geo/GeoJson.cpp

```diff
@@ -238,36 +302,51 @@ Result parseLineImpl(velocypack::Slice vpack, std::vector<S2Point>& vertices) {
 
 template<bool Validation>
 Result parseLinesImpl(velocypack::Slice vpack, std::vector<S2Polyline>& lines,
-                      std::vector<S2Point>& vertices) {
+                      std::vector<S2LatLng>& vertices, coding::Options options,
+                      Encoder* encoder) {
   TRI_ASSERT(vpack.isArray());
   velocypack::ArrayIterator it{vpack};
+  [[maybe_unused]] auto const n = it.size();
```

Comment:
Now `n` is always used in `lines.reserve(n);`

Reply:
Fixed